### PR TITLE
Add a 'Share with friends' option after form submission

### LINKED ISSUE #106: Add a 'Share with friends' option after form submission & update success message copy
1) Once users successfully submit a form on /update, below the success message we want to add a Share App CTA option that allows users to share the app on their social networks. 

There will be pre-filled copy that needs to be submitted with the share button - 
"Covid Maps - find essentials services around you in the lockdown period - track timings, stock levels and safety precautions at stores. Make an update with info you find in your grocery run and share with neighbours! - https://covidmaps.in/"

Sharing options to include- WhatsApp, Messenger, Instagram, SMS, Copy to ClipBoard, Twitter, GMail

2) Update Success Message copy to - "And now we're up-to-date! Thanks for keeping them coming 🙌"

### PR DESCRIPTION
**Pull request by eragon512**
- Add a 'Share with friends' option after form submission
- Change Submission/Update Success Message Copy

Fixes #106

### CODE REVIEW

**Comment by arjun27** [comment]
hey @eragon512 is this ready for a review? @Siddharth11 is picking up some related stuff, and i wanted to see if we can land this

**Comment by eragon512** [comment]
> hey @eragon512 is this ready for a review? @Siddharth11 is picking up some related stuff, and i wanted to see if we can land this

yeah, its ready for a review

File: src/SubmitPage/SubmitForm.js [hunk 1]
```diff
@@ -166,7 +167,23 @@ class SubmitForm extends React.Component {
...
+                title='Covid Maps'
+                url='https://covidmaps.in/'
+                text={
+                  [
+                    'Covid Maps - find essentials services around you in the lockdown period ',
+                    '- track timings, stock levels and safety precautions at stores. ',
+                    'Make an update with info you find in your grocery run and share with neighbours! ',
+                    '- https://covidmaps.in/',
+                  ]
+                    .join()
```
Comment (Siddharth11) [inline src/SubmitPage/SubmitForm.js:183]: What's the reason to break the text into 3 parts and then rejoin them ?

**Comment by Siddharth11** [comment]
@arjun27 @asanjeevrao @eragon512 i'm merging this PR in a separate branch. Because of new changes in master, I've to test these changes locally. Then I'll reopen a new PR for this

**Pull request merged**